pkg/charts/utils.go

```diff
@@ -193,9 +190,8 @@ func generateChartValues(config *ciliumv1alpha1.NetworkConfig, network *extensio
 
 	// If ETCD enabled
 	if config.Store != nil {
-		if *config.Store == ciliumv1alpha1.ETCD {
-			globalConfig.Etcd.Enabled = true
-			globalConfig.Etcd.Managed = true
+		if *config.Store == ciliumv1alpha1.Kubernetes {
+			return requirementsConfig, globalConfig, fmt.Errorf("%s is not a supported value for field store", *config.Store)
```

Comment:
Should this really be an equal check? I think we always use kubernetes as backing store. Hence, anything not equal to kubernetes would be not supported...

Reply:
Reversed the condition.